scss/6-components/_callout.scss

```diff
@@ -119,3 +123,10 @@
     margin-top: $cads-spacing-4;
   }
 }
+
+.cads-prose .cads-callout__title {
```

Comment:
Guessing this is because callouts mainly appear in body content? Makes sense, but could we add an explanatory comment?